connexion/apps/flask_app.py

```diff
@@ -19,9 +19,12 @@
 class FlaskApp(AbstractApp):
     def __init__(self, import_name, server='flask', **kwargs):
         super(FlaskApp, self).__init__(import_name, FlaskApi, server=server, **kwargs)
-
+        self.flask_dict={}
```

Comment:
kwargs is already a dict.
why not `self.flask_args = kwargs`?

Reply:
just make sure the kwargs parameters, I am not deeply in this connexion, afraid  **kwargs change, like: 
```
kwargs.pop('foo')
```
in your oldstyle or AbstractApp initialization.

Reply:
for future reference, in that case you can use `copy`